modules/bitgo/test/v2/unit/tss/helpers.ts

```diff
@@ -19,3 +20,28 @@ export async function nockGetTxRequest(params: {walletId: string, txRequestId: s
     .get(`/api/v2/wallet/${params.walletId}/txrequests?txRequestIds=${params.txRequestId}&latest=true`)
     .reply(200, params.response);
 }
+
+export async function createWalletSignatures(
+  privateKeyArmored: string,
+  publicKeyToCertArmoredUser: string,
+  publicKeyToCertArmoredBackup: string,
+  notations: { name: string; value: string }[]
+): Promise<string> {
+  const certsUserString = await createSharedDataProof(privateKeyArmored, publicKeyToCertArmoredUser, notations);
+  const certsBackupString = await createSharedDataProof(privateKeyArmored, publicKeyToCertArmoredBackup, notations);
+
+  const certsUserKey = await openpgp.readKey({ armoredKey: certsUserString });
+  const certsBackupKey = await openpgp.readKey({ armoredKey: certsBackupString });
+
+  const finalKey = new openpgp.PacketList();
```

Comment:
nit: rename to something like 'mergedWalletKeys` or `mergedWalletSigs`.   because this is not a one key, but rather is merging two pgp keys into one armored blob.

Reply:
Done